extensions/send-to-flomo/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "send-to-flomo",
+  "title": "Send to flomo",
+  "description": "Raycast extension for sending MEMOs to flomo",
+  "icon": "command-icon.png",
+  "author": "ahonn",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "send-to-flomo",
+      "title": "Send MEMO",
+      "subtitle": "flomo",
+      "description": "Send your thoughts to flomo",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "api",
+      "title": "MEMO API URL",
+      "description": "Your MEMO API URL",
+      "type": "textfield",
+      "required": true
+    },
+    {
+      "name": "defaultTags",
+      "title": "Default tag(s)",
+      "description": "The default tag(s) that comes with each MEMO — separated by space",
```

Comment:
```suggestion
      "title": "Default tags",
      "description": "The default tags that comes with each MEMO — separated by space",
```

My bad, this sounds better 😄

Reply:
Agree, I updated this command description